package.json

```diff
@@ -26,6 +26,7 @@
     "core-js": "^3.15.1",
     "gsap": "^3.8.0",
     "jsonp": "^0.2.1",
+    "luxon": "^2.4.0",
```

Comment:
Note: Using luxon for now because it's what we use on the airdrop site. We can switch to something later, if needed.